app/views/actuators/index.html.haml

```diff
@@ -1,5 +1,17 @@
-%h1 Listing actuators
+%h1 Actuators
+%hr
+%strong Actuators are devices connected to a port at reporter.box. The reporter.box can switch them on and off. This ist done with chains. These are all your actuators:
+%hr
+%h3 Manage Actuators
+%strong Do you have a new device you would like to connect to the Sensor-Live-Report? Create a new actuator!
+%p
+=link_to new_actuator_path, {:class => "btn btn-primary btn-lg"} do
+  %span.glyphicon.glyphicon-plus-sign
```

Comment:
@drjakob `glyphicons` are not displayed correctly. The [twitter-bootstrap-rails](https://github.com/seyhunak/twitter-bootstrap-rails) gem uses [font awesome icons ](http://fontawesome.io/icons/)per default. I prefer defaults. You can however override [our configuration](https://github.com/roschaefer/story.board/blob/master/app/assets/stylesheets/bootstrap_and_overrides.css#L5) to use bootstrap glyphicons.